plugins/scaffolder-backend-module-confluence-to-markdown/config.d.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /** Configuration options for the Confluence to Markdown action */
+  confluence?: {
+    /**
+     * The base URL for accessing the Confluence API
+     */
+    baseUrl: string;
+    /**
+     * The authentication token for accessing the Confluence API
+     * * @visibility secret
```

Comment:
double asterisk here, i wonder if that actually would risk NOT marking it as secret 🤔 

I think it would make sense to document here too what the two supported token formats are, at least just very briefly.